python/GangaPanda/PACKAGE.py

```diff
@@ -8,7 +8,7 @@
 """
 
 _external_packages = {
-    'panda-client' : { 'version' : '0.5.69',
+    'panda-client' : { 'version' : '0.5.80',
                     'syspath':['lib/python2.6/site-packages'],
                     'CONFIGEXTRACTOR_PATH':'etc/panda/share',
                     'PANDA_SYS':'.',
```

Comment:
This is because, historically, we have both GangaAtlas and GangaPanda modules (one for the general Atlas code and one for the grid/Panda backend) and in the dim and distant past you didn't need both loaded to run jobs (thinking about that's technically still true actually 😄 ). However, this is definitely something that is ripe for clean up in the future!